### DISCUSSION TITLE
Is there an easy way to remove the upload attachment option from the input widget?

### DISCUSSION THREAD

**Discussion by itaishalom** [flyerhq/flutter_chat_ui#127] [Q&A] [answered]
As of right now, I don't want an upload option,
Is there an easy way to remove it? I don't want to implement the bottom widget from scratch.
From viewing the code, I see it's not supported right now. Can I PR?
Thanks Alex.

**Comment by itaishalom** [comment]
You know what, I think I'll implement it all by myself.
Thanks anyway!

**Comment by demchenkoalex** [comment, marked as answer]
It was supported, just don't pass `onAttachmentPressed` :)

**Reply by itaishalom** [reply]
Cool, thanks man!

**Comment by itaishalom** [comment]
There is an issue though, if I use an input text decoration like this:

```
InputDecoration inputTextDecoration = const InputDecoration(
filled: true,
fillColor: Color(0xff060606),

  border: OutlineInputBorder(
    borderSide: BorderSide.none,
    borderRadius: BorderRadius.all(Radius.circular(20.0)),
  ),
  contentPadding:
  const EdgeInsets.only(left: 14.0, bottom: 8.0, top: 8.0),
  isCollapsed: false,
),
```
There is an unexpected behavior with the send button which is changing the size of the input textfiled.
What do you think?

https://www.youtube.com/watch?v=VbfedxKyJpQ

**Reply by demchenkoalex** [reply]
Hmm AFAIR there is a property to keep it always on, will that help? Cause I'm not sure it is possible to have it inside the input

**Reply by itaishalom** [reply]
I ended up copying 80% of the Input widget and changed the textfield to contain a suffixWidget,
that solved my problem.
Keep up the good work!
